script_library/requirements.txt

```diff
@@ -0,0 +1,4 @@
+ansible==2.7.5
+netaddr==0.7.19
+jmespath==0.9.3
+requests==2.21.0
```

Comment:
same here - wouldn't any requests version work?

Reply:
Most likely.

See also previous comment on the reason of the venv, and their freezing.